### ISSUE TITLE
RGB OME-TIFF and Blending

### ISSUE THREAD

**Issue by MosGeo** [napari/napari-tiff#38]
I am generating a simple RGB OME-TIFFS. Everything is working as expected with saving the file using tifffile and napari-tiff is recognizing them as RGB images (`PHOTOMETRIC.RGB`).

The issue is that napari-tiff defaults to adding blending metadata as "additive" even for a simple RGB image. This means opening two of them in napari will mess things up. This is not the expected behaviour. The expected blending in this case is "translucent_no_depth". As far as I can see, this is not related to the ome-tiff file metadata and it is just related to how napari-tiff decides what blending to use.

The relevent line is:
https://github.com/napari/napari-tiff/blob/8c704181275bc21e2ba220e4bd1bf44a97b204c4/napari_tiff/napari_tiff_metadata.py#L340

**Comment by DragaDoncila** [comment]
Pinging @Czaki @jni as they have more experience with this repo - any guidance here?

**Comment by Czaki** [comment]
When I wrote my multichannel reader, I always set the first layer to translucent and the rest to additive. 
As in this case, we have single RGB so translucent is more expected.

**Comment by jni** [comment]
I agree with the symptom and the diagnosis. @MosGeo are you able to open a PR to fix? I think @Czaki's suggestion (first layer in the same reader call is translucent, additional layers are additive) is correct.

Aside: translucent_no_depth is for 3D rendering only, it means e.g. a bright pixel in the image (displayed in MIP mode) will not hide a point that is further away from the camera. For 2D rendering they are the same.

**Comment by MosGeo** [comment]
@jni just got arround playing with this:

First, regarding RGB images, I think it is important to distingush between SamplesPerPixels (which is 3 in the case of RGB) and Channel (which is 1 in the case of RGB). The other important aspect is the `PHOTOMETRIC.RGB` flag in OME-TIFF metadata.

Based on these definitions, an RGB channel is one where we have 3/4 samples per pixel and the photometric.rgb flag is set (without relying on magic, e.g., assuming it is RGB if we have 3 channels even if photometric.rgb flag is wrong).

That being said, Let's discuss some possibilities:

**1. One channel grayscale image**
I don't have a preference but in my mind, the default should be translucent.

**2. Multiple channels gray scale image**
I think this is the case where additive is expected and multiple layers should be used?

**3. One channel RGB image**
I expect that the image will be loaded as 1 translucent layer.

**4. Multiple channels and each channel is RGB (I deal with images of this nature):**
In my use case, my preference is that the image is loaded as translucent and as 1 layer only (because the images are colocated and it is easy to switch between them using the slider or anlyze them as a group). That being said I see the argument that it should be multiple layers and each one is translucent. Do you have use cases where it should be additive here?

**5. Multiple mix of channels and each channel is either grayscale or RGB.**
I don't have use case for this. My mind tells me translucent with each channel (be it RGB or grayscale) as a seperate layer.

Ho do you want to handle each case? I can give it a go once it is decided.

**Comment by MosGeo** [comment]
@jni let's follow napari's convention/standard for the plugin instead of my special case. I will make the changes in my branch and do a PR for review.

p.s. yah, napari can be improved in layer perspective. For example, i wish i can 1) group layers and quickly switch between them/hide-unhide them, 2) have children layers (for example, have the semenatic segmentation result) as a child layer of the input or have the roi layer for the plugin as a child layer of the input layer). Also, I wish i can lock layer (at some point i started implemented that and opened a discussion). I should get back to it now that i know more about napari structure and python (e.g., I remember it was crashing with napari evented models or something).
